escrow/project/contracts/escrow-contract/tests/functions/core/accept_arbiter.rs

```diff
@@ -88,50 +87,56 @@ mod success {
             &arbiter_obj,
             &defaults.asset_id,
             vec![asset.clone(), asset.clone()],
-            buyer.wallet.address(),
-            &seller.contract,
+            &buyer,
+            &seller,
             defaults.deadline,
         )
         .await;
 
+        propose_arbiter(arbiter_obj2.clone(), &seller, 0).await;
+        propose_arbiter(arbiter_obj2.clone(), &seller, 1).await;
+
+        assert_eq!(2, asset_amount(&defaults.asset_id, &seller).await);
```

Comment:
```suggestion
        assert_eq!(defaults.asset_amount, asset_amount(&defaults.asset_id, &seller).await);
```
not sure if this is correct, but it seems like we should be able to get this number from a variable

Reply:
The suggestion won't work. I forgot to update this part. Proposing an arbiter has a cost. Since arbiter 2 has a `-1` doing that twice results in `2` being left over in the wallet.